Problem with package build on windows

I am trying to setup helix with the runtime and fastly but it doesn't build properly. There are no errors in logs. There is only one zip package created static.zip, html.zip is missing.
I tried to build it on Windows 10 with:
node: 10.15.0
npm: 6.4.1
helix:  2.1.6 and o.14.12

Here are the steps:
1. Create empty repo on github
2. helix demo helix-demo
3. git remote add origin
4. git push origin master
5. hlx deploy with  --adddefault
6. hlx publish

It looks like it is deployed correctly to fastly, running wsk list returns following actions deployed:
hlx--static (only one)

When i invoke an action i get following error:
` "error": "Initialization has failed due to: Error: Cannot find module 'request-promise-native'\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:507:25)\n    at Module.require (internal/modules/cjs/loader.js:637:17)\n    at require (internal/modules/cjs/helpers.js:22:18)\n    at Object.<anonymous> (/nodejsAction/XXMIjDJm/static.js:13:17)\n    at Module._compile (internal/modules/cjs/loader.js:689:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)\n    at Module.load (internal/modules/cjs/loader.js:599:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:530:3)"`

logs are empty.

Is it a known issue?

Looks better now, there is html.zip and i have new action deployed to the runtime:
/pid80102/github-com--wielmic--hlx-test--master-dirty/html             private nodejs:10

but when i test it through fastly (test domain) i get an error in the browser

`{
  "code": "SRdzzwytzkWqwDpEriixF89YUQYKdhzt",
  "error": "There was an error processing your request."
}` 
and then when i check via wsk activation get i get:

'
            "error": "Initialization has failed due to: Error: Cannot find module 'request-promise-native'\n    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)\n    at Function.Module._load (internal/modules/cjs/loader.js:507:25)\n    at Module.require (internal/modules/cjs/loader.js:637:17)\n    at require (internal/modules/cjs/helpers.js:22:18)\n    at Object.<anonymous> (/nodejsAction/XXcFnIne/static.js:13:17)\n    at Module._compile (internal/modules/cjs/loader.js:689:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)\n    at Module.load (internal/modules/cjs/loader.js:599:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:530:3)"`

which is the initial error. 
From wsk activation list it looks like only hlx--static:0.0.7 was called

--- a/src/package.cmd.js
+++ b/src/package.cmd.js
@@ -135,7 +135,7 @@ class PackageCommand extends StaticCommand {
         // if the module was linked via `npm link`, then it is a checked-out module, and should
         // not be included as-is.
         const modPath = info.externals[mod];
-        if (modPath.indexOf('/node_modules/') < 0) {
+        if (modPath.indexOf('/node_modules/') < 0 || modPath.indexOf('\\node_modules\\') < 0) {
           // todo: async
           // todo: read .npmignore
           const files = glob.sync('!(.git|node_modules|logs|docs|coverage)/**', {

--- a/src/parcel/ExternalsCollector.js
+++ b/src/parcel/ExternalsCollector.js
@@ -63,7 +63,7 @@ class ExternalsCollector {
     // todo async
     const pkgInfos = {};
     function resolveByPackageJson(resource) {
-      if (!resource || resource === '/') {
+      if (!resource || resource === '/' || path.dirname(resource) === resource) {
         return [];
       }
       let info = pkgInfos[resource];

